obspy/imaging/beachball.py

```diff
@@ -279,7 +283,7 @@ def beachball(fm, linewidth=2, facecolor='b', bgcolor='w', edgecolor='k',
         return fig
 
 
-def plot_mt(T, N, P, size=200, plot_zerotrace=True,  # noqa
+def plot_mt(T, N, P, size=200, plot_zerotrace=False,  # noqa
```

Comment:
not an expert here, but this changes the default behaviour of a public def, not sure it's ideal/desired?